essay.tex

```diff
@@ -0,0 +1,61 @@
+
+\section{The research}
+One of the central idea's suggested by Cockburn and Highsmith \cite{AgilePeople} is that Agile helps to improve motivation and morale, by implementing increased communication and reduced documentation. This helps to build a sense of community within the team which then in turn improves their morale.
+
+Looking at the work of Asproni \cite{MotivationSoftware} he explores the key motivational factors for software developers. Drawing the conclusion that a developer is more interested in the work being interesting, than being paid more. It's not difficult to assume that people working within the games industry being interested in their work is a huge benefit and will help to improve their enthusiasm for the project. 
+
+Research carried out by McHugh, Conboy and Lang \cite {AgilePractices} shows some interesting results in which they spoke with already established teams, allowing them skip over as Tuckman \cite {Tuckman} suggests in his work the initial stages of Forming, Norming, etc. Reducing the possible effects on the teams motivations that can occur. In their research they found that many of the people who had experience from a more traditional management style, had little desire to work on a team that did not use an Agile approach.
+ 
+The research from various authors shows that the key to productive teams is: to make sure you build a sense of community, to make sure that the team are involved in work they find interesting and to let them experience other types of management so they can decide which is best.
+
+\section{Agile in the Gaming Industry}
+
+The adaptive nature of Agile means that it's well suited for software development. Some of the reflective work from software development industry its self \cite{Practitioners}, leads credence to the above research. However when looking within the gaming industry not all aspect of Agile can be considered the best option \cite{RyseCrisis} a reflective look at Agile development mainly on Scrum revealed that Agile was very useful at the initial stages of development, but as time progressed the need for a more structured or linear approach was necessary to ensure that deadlines were met. Though later expands on this point to say that being able to change to a different style of development could also be considered Agile in it's own way. 
+
+There are many useful studies which have taken place for both the software and management side of using Agile, however the gaming industry combines several disciplines which need to co-operate. I have been unable to find much in the way of quantified research upon the topic of motivation and morale within a game development company and the effects Agile has had upon them. I believe a further study into the subject matter would be prudent, especially considering the huge implications of motivation on a teams productivity.
```

Comment:
Simply state that `there is limited research specific to the games industry`. Also, include a *but* and exercise your own intellect a little. Infer something. Synthesis something from the literature on other industries and compare it to your knowledge of games development.